internal/codegen/tpl_stencil_test.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2022 Outreach Corporation. All Rights Reserved.
+
+// Description: This file contains the public API for templates
+// for stencil
+
+package codegen
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestTplStencil_ReadBlocks(t *testing.T) {
+	type fields struct {
+	}
+	type args struct {
+		fpath string
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		args    args
+		want    map[string]string
+		wantErr bool
+	}{
+		{
+			name: "should read blocks from a file",
+			args: args{
+				fpath: "testdata/blocks-test.txt",
+			},
+			want: map[string]string{
+				"helloWorld": "Hello, world!",
+				"e2e":        "content",
+			},
+		},
+		{
+			name: "should error on out of chroot path",
+			args: args{
+				fpath: "../testdata/blocks-test.txt",
+			},
+			wantErr: true,
```

Comment:
Given that this is a security check, it would be better if you verified the error type / contents and not just that one was returned.

Reply:
Hm yeah I can do that. These test were 99% generated by VSCode so it doesn't support that OOTB but I can quickly edit the table tests to support checking the error type / message (if the type isn't available)